generate.go

```diff
@@ -30,11 +30,11 @@ func (c Cfg) GenerateValue() string {
 	if c.encrypted {
 		// TODO COGS-1657
 		// decrypt.File(c.Path, c.SubPath)
-		return "|enc| " + c.Path
+		return "|enc|" + c.Path
```

Comment:
In case you wanted to nerd out, I [found this](https://dev.to/pmalhaire/concatenate-strings-in-golang-a-quick-benchmark-4ahh) over the weekend on the different ways to combine strings in Golang, apparently shows that using `+` over the `fmt` package is much better.

Nothing to change here, just wanted to share